purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -0,0 +1,85 @@
+package com.revenuecat.purchases
+
+import android.content.Context
+import java.util.concurrent.ExecutorService
+
+open class PurchasesConfiguration(builder: Builder) {
+
+    val context: Context
+    val apiKey: String
+    val appUserID: String
+    val observerMode: Boolean
+    val service: ExecutorService?
+    internal val store: Store
+    internal val diagnosticsEnabled: Boolean
+    val dangerousSettings: DangerousSettings
+    val verificationMode: EntitlementVerificationMode
+
+    init {
+        this.context = builder.context
+        this.apiKey = builder.apiKey
+        this.appUserID = builder.appUserID
+        this.observerMode = builder.observerMode
+        this.service = builder.service
+        this.store = builder.store
+        this.diagnosticsEnabled = builder.diagnosticsEnabled
+        this.verificationMode = builder.verificationMode
+        this.dangerousSettings = builder.dangerousSettings
+    }
+
+    open class Builder(
```

Comment:
Note that I removed the `observerMode` setter as well, so it won't be available in the customEntitlementComputation package.